hadoop/src/main/java/org/apache/carbondata/hadoop/CarbonInputFormat.java

```diff
@@ -133,7 +133,7 @@ public static void setTableInfo(Configuration configuration, TableInfo tableInfo
   /**
    * Get TableInfo object from `configuration`
    */
-  private TableInfo getTableInfo(Configuration configuration) throws IOException {
+  public TableInfo getTableInfo(Configuration configuration) throws IOException {
```

Comment:
In the future,  carbondata will use CarbonTableInputFormat to replace CarbonInputFormat, so please use CarbonTableInputFormat.

Reply:
@cenyuhai change in this file is not required now  because MapredCarbonInputFormat now extends CarbonTableInputFormat